app/src/main/res/values/strings.xml

```diff
@@ -100,7 +100,7 @@
     <string name="settings_key_lock_barcode_orientation" translatable="false">pref_lock_barcode_orientation</string>
     <string name="settings_keep_screen_on">Keep screen on while viewing a card</string>
     <string name="settings_key_keep_screen_on" translatable="false">pref_keep_screen_on</string>
-    <string name="settings_disable_lockscreen_while_viewing_card">Disable lock screen while viewing a card</string>
+    <string name="settings_disable_lockscreen_while_viewing_card">No lock screen while displaying cards</string>
```

Comment:
```suggestion
    <string name="settings_disable_lockscreen_while_viewing_card">No lock screen while displaying</string>
```

Reply:
Definitely not an improvement, as the lock screen is only disabled while viewing a card. So now it is no longer clear what someone is changing.